R/NAMESPACE

```diff
@@ -0,0 +1,39 @@
+# Generated by roxygen2: do not edit by hand
+
+S3method("$",metaflow.flowspec.FlowSpec)
+S3method("$<-",metaflow.flowspec.FlowSpec)
+S3method("[[",metaflow.flowspec.FlowSpec)
+S3method("[[<-",metaflow.flowspec.FlowSpec)
+export("%>%")
+export(current)
+export(decorator)
+export(dict)
+export(flow_client)
+export(gather_inputs)
+export(get_namespace)
+export(list_flows)
+export(merge_artifacts)
+export(metaflow)
+export(metaflow_location)
+export(mf_client)
+export(new_flow)
+export(new_run)
+export(new_step)
+export(new_task)
+export(parameter)
+export(py_to_r)
+export(r_to_py)
+export(run)
+export(run_client)
+export(set_default_namespace)
+export(set_namespace)
```

Comment:
We would need to support `set_metadata` and `get_metadata` as well.

Reply:
Good catch. Adding these two methods.